app/assets/stylesheets/views/_topical_events.scss

```diff
@@ -90,4 +90,8 @@
     }
     /* stylelint-enable */
   }
+
+  .topical-events__margin-bottom {
+    margin-bottom: govuk-spacing(6);
+  }
```

Comment:
This seems a somewhat unusual class. By [BEM](http://getbem.com/naming/) this would be describing the margin-bottom element of a topical-events object. But we neither have a margin-bottom element or a topical-events object.

I think it'd be better to just the govuk-!-margin modifiers directly to be honest, as a simple suggestion. As I guess we want just a utility class and we have that in the govuk- option but without the risks of this utility class which will couple any changes to affect multiple things (which makes the components not isolated)

Reply:
I was taking inspiration from [this existing css for organisations](https://github.com/alphagov/collections/blob/main/app/assets/stylesheets/views/_organisation.scss#L83-L85) in the repo, and trying to fix the issue where we were using a lot of different values for margin-bottom.

But I also don't really know much about frontend practice so maybe this doens't make sense here, happy to change

Reply:
Ah sure I didn't realise we had that for organisations. It'd be good to reference that in the commit message.

From Andy's comment it seems like he is happy with it https://github.com/alphagov/collections/pull/2853#issuecomment-1187475459 so by that logic and precedence it seems ok to keep.

It makes me personally a bit sad as it's not using BEM conventions correctly but not a hill I'm willing to die on, particularly as there is precedence.